src/main/java/org/alliancegenome/curation_api/model/entities/VocabularyTerm.java

```diff
@@ -26,7 +26,7 @@
 @Schema(name="VocabularyTerm", description="POJO that represents the Vocabulary Term")
 public class VocabularyTerm extends BaseGeneratedEntity {
 
-    @FullTextField(analyzer = "autocompleteAnalyzer", searchAnalyzer = "autocompleteSearchAnalyzer")
+    @FullTextField(analyzer = "whitespace", searchAnalyzer = "whitespace")
```

Comment:
Curious the reason for this change? What are autocomplete Analyzers not doing that they should?

Reply:
Ahh ok... I didn't think about the _'s in the terms